doc/nrf/releases_and_maturity/releases/release-notes-2.9.0.rst

```diff
@@ -0,0 +1,661 @@
+    * Wait until a triggered Bluetooth LE connection parameters update is completed before triggering subsequent updates for a given connection.
+    * Improve the log to also display the information if USB is suspended.
+      The information is needed to determine the requested connection parameters.
+    * Use non-zero Bluetooth LE peripheral latency while USB is suspended.
+      This is done to prevent peripheral latency increase requests from :ref:`nrf_desktop_ble_latency` on peripheral's end.
+    * Revert the USB suspended Bluetooth LE connection parameter update when USB cable is disconnected.
+
+  * The :ref:`nrf_desktop_ble_scan` to always use a connection interval of 10 ms for peripherals without Low Latency Packet Mode (LLPM) support if a dongle supports LLPM and more than one Bluetooth LE connection.
+    This is required to avoid Bluetooth Link Layer scheduling conflicts that could lead to HID report rate drop.
+
+* Removed imply for partial erase feature of the nRF SoC flash driver (:kconfig:option:`CONFIG_SOC_FLASH_NRF_PARTIAL_ERASE`) for the USB next stack (:ref:`CONFIG_DESKTOP_USB_STACK_NEXT <config_desktop_app_options>`).
+  The partial erase feature was used as a workaround for device errors that might be reported by the Windows USB host in Device Manager if a USB cable is connected while erasing a secondary image slot in the background.
+  The workaround is no longer needed after the nRF UDC driver was improved.
+
+Samples
+=======
+
+This section provides detailed lists of changes by :ref:`sample <samples>`.
+
+Bluetooth samples
+-----------------
+
+* Added:
+
+  * The :ref:`channel_sounding_ras_reflector` sample demonstrating how to implement a Channel Sounding Reflector that exposes the Ranging Responder GATT Service.
+  * The :ref:`channel_sounding_ras_initiator` sample demonstrating Channel Sounding by setting up a Channel Sounding Initiator that acts as a Ranging Requestor GATT Client.
+    It includes a basic distance estimation to demonstrate IQ data handling.
+    The accuracy is not representative for Channel Sounding and should be replaced if accuracy is important.
+  * The :ref:`bt_peripheral_with_multiple_identities` sample demonstrating how to use a single physical device to create and manage multiple advertisers, making it appear as multiple distinct devices by assigning each a unique identity.
+  * The :ref:`bt_scanning_while_connecting` sample demonstrating how to establish multiple connections faster using the :kconfig:option:`CONFIG_BT_SCAN_AND_INITIATE_IN_PARALLEL` Kconfig option.
+  * Support for the ``nrf54l15dk/nrf54l05/cpuapp`` and ``nrf54l15dk/nrf54l10/cpuapp`` board targets in the following samples:
+
+    * :ref:`direct_test_mode`
+    * :ref:`peripheral_hids_mouse`
+    * :ref:`peripheral_lbs`
+    * :ref:`power_profiling`
+    * :ref:`peripheral_uart`
+
+* Updated:
```

Comment:
Hi @peknis, probably this entire entry  should be moved back to the changelog, it is relevant for 2.9-nRF54H20.

Reply:
Kepth in both.